### ISSUE TITLE
Vue version mismatch in nuxt

### ISSUE THREAD

**Issue by walleXD** [rafaelpimpa/buefy#64]
Hi all,
Given that buefy has Vue as one of its dependencies, it installs Vue 2.3.3 but nuxt currently uses Vue 2.2.6. This leads to the following issue
```
Vue packages version mismatch:

- vue@2.3.3
- vue-server-renderer@2.2.6
```
This continues on for all other vue associated packages. Is there any way to disentangle the vue install with the package and use the one installed with nuxt.js

**Comment by rafaelpimpa** [comment]
I think you should update [vue-server-renderer](https://www.npmjs.com/package/vue-server-renderer) it's version always associate with vue itself.

**Comment by diit** [comment]
`vue-server-renderer` version is set by nuxt and the rc11 version (Current) leads to the above error. 

IMO this issue should be reopened as the buefy library is unusable with nuxt currently.

**Comment by wopian** [comment]
`vue` should ideally be a `peerDependency`, which will use the `vue` version in the user's package.json 👍

**Comment by microcipcip** [comment]
I fixed it with `yarn add vue@2.5.13 vue-template-compiler@2.5.13 vue-server-renderer@2.5.13`
